app/models/miq_worker/runner.rb

```diff
@@ -319,8 +319,7 @@ def config_out_of_date?
   end
 
   def key_store
-    require 'dalli'
-    @key_store ||= Dalli::Client.new(MiqMemcached.server_address, :namespace => "server_monitor")
+    @key_store ||= MiqMemcached.client("server_monitor")
```

Comment:
Should this be `:namespace => "server_monitor"` since the caller takes the options hash?